jOOR-java-8/src/main/java/org/joor/Compile.java

```diff
@@ -106,47 +110,6 @@
                     result = fileManager.loadAndReturnMainClass(className,
                         (name, bytes) -> Reflect.on(cl).call("defineClass", name, bytes, 0, bytes.length).get());
                 }
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
```

Comment:
There was a reason for this whitespace:
https://blog.jooq.org/2016/03/01/how-to-support-java-6-8-9-in-a-single-api/

Please do not remove it.